EssentialEstablishmentGenerator/Tavern/js/createTavern.js

```diff
@@ -76,8 +76,24 @@ setup.createTavern = function (town, opts) {
   tavern.lodging = ''
   tavern.sin = ''
   tavern.food = ''
+  tavern.colour1 = [setup.colours.yellow.colour.seededrandom(), setup.colours.orange.colour.seededrandom(), setup.colours.red.colour.seededrandom(), setup.colours.purple.colour.seededrandom(), setup.colours.blue.colour.seededrandom(), setup.colours.green.colour.seededrandom(), setup.colours.brown.colour.seededrandom(), setup.colours.black.colour.seededrandom(), setup.colours.white.colour.seededrandom()].seededrandom()
+  tavern.colour2 = [setup.colours.yellow.colour.seededrandom(), setup.colours.orange.colour.seededrandom(), setup.colours.red.colour.seededrandom(), setup.colours.purple.colour.seededrandom(), setup.colours.blue.colour.seededrandom(), setup.colours.green.colour.seededrandom(), setup.colours.brown.colour.seededrandom(), setup.colours.black.colour.seededrandom(), setup.colours.white.colour.seededrandom()].seededrandom()
   tavern.bedCleanliness = ''
-
+  // Define entertainment if large enough
```

Comment:
Rather than separating via concrete cutoffs, we could make another roll, `fairmath`'d to weight it properly.

`let entertainment = Math.fm((tavern.roll.wealth - 50), random(1, 100)`
That should give a relatively stable thing, but still yield the occasional outlier.

Reply:
I'm not quite sure what this request, as in if the tavern is too small it will occasionally still roll entertainment or?